nebula-logger/main/logger-engine/classes/ComponentLogger.cls

```diff
@@ -0,0 +1,269 @@
+//------------------------------------------------------------------------------------------------//
+// This file is part of the Nebula Logger project, released under the MIT License.                //
+// See LICENSE file or go to https://github.com/jongpie/NebulaLogger for full license details.    //
+//------------------------------------------------------------------------------------------------//
+
+/**
+ * @group Logger Engine
+ * @description Controller classs used by the light web component `c/logger`
+ * @see Logger
+ * @see LogEntryEventBuilder
+ */
+public inherited sharing class ComponentLogger {
+    @TestVisible
+    private static final String EXCEPTION_TYPE_PREFIX = 'JavaScript.';
+
+    /**
+     * @description Provides data to the frontend about `LoggerSettings__c` & server-supported logging details
+     * @return   return The instance of `ComponentLoggerSettings` for the current user
+     */
+    @AuraEnabled(cacheable=true)
+    public static ComponentLoggerSettings getSettings() {
+        return new ComponentLoggerSettings();
+    }
+
+    /**
+     * @description saveComponentLogEntries Saves log entries created via lwc or aura components
+     * @param  componentLogEntries The list of `ComponentLogEntry` objects to save via `Logger`
+     * @return                     return The transaction ID (based on `Logger.getTransactionId())`
+     */
+    @AuraEnabled
+    public static String saveComponentLogEntries(List<ComponentLogEntry> componentLogEntries) {
+        System.debug('componentLogEntries=' + componentLogEntries);
```

Comment:
are you planning to keep this?

Reply:
Nope! This was just for testing, I forgot to remove it - thanks for pointing it out!